tests/src/JIT/Performance/CodeQuality/Layout/SearchLoops.cs

```diff
@@ -47,13 +47,13 @@ public SearchLoops()
         }
 
         [Benchmark(InnerIterationCount = 20000000)]
-        public void LoopReturn()
+        public void LoopReturnIter()
```

Comment:
@sbomer Namespace, type name, method name and argument names are part of benchmark ID. When you change it, you are introducing a new benchmark.

This change has introduced new benchmark to BenchView.

Please don't rename benchmarks in the future.

/cc @jorive @AndyAyersMS

Reply:
Noted, thanks. Just FYI, these were renamed to satisfy the xunit analyzer in the new xunit runner - it didn't like benchmarks methods that had an overload with the same name. The analyzer should enforce this for new benchmarks going forward.